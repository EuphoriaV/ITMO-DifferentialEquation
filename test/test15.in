4.88
(((x * (x - 3.09)) * x) * (6.67 - ((x * x) * x)))
1.44
5.81