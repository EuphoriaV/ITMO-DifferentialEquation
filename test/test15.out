0.0038204